src/bitbucket/BitbucketClient.ts

```diff
@@ -60,22 +62,31 @@ export class BitbucketClient {
 
     if (!approvalChecks.isOpen) {
       errors.push('PR is already closed!');
-    } else if (prSettings.customChecks) {
+    } else {
       const pullRequestInfo = {
         pullRequest,
         buildStatuses,
         approvals,
         permissionLevel,
       };
-      for (const { rule } of prSettings.customChecks) {
-        const passesRule = await rule(pullRequestInfo);
-        if (typeof passesRule === 'string') errors.push(passesRule);
+      if (prSettings.customChecks) {
+        for (const { rule } of prSettings.customChecks) {
+          const passesRule = await rule(pullRequestInfo, axios);
+          if (typeof passesRule === 'string') errors.push(passesRule);
```

Comment:
Hmmm.. I get why you added the axios injection, but it still feels a bit iffy to me...
Lets go with it for now and see if we need to change it later I guess.

I think in the end, if you config needs external packages, you should just have them installed. But lets go with this for now